google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -103,6 +103,8 @@ public Void apply(WriteLogEntriesResponse input) {
         }
       };
   private static final ThreadLocal<Boolean> inWriteCall = new ThreadLocal<>();
+  private static final boolean FLUSH_ON_ERROR = Boolean.parseBoolean(System.getenv("FLUSH_ON_ERROR"));
+  private static final int BUFFER_SIZE = 1000;
```

Comment:
This should also be configurable from somewhere, If not settings then via env variable